README.md

```diff
@@ -43,11 +43,13 @@ Enter the virtual environment as described above, then:
 ## How to post data.
 To post data, use /send endpoint with POST method. Data must be in the following format.
 
- {
-    "type": "TEST_MAGIC_LINK",
-    "to": "test_recipient@email.com",
-    "content": "MAGIC LINK GOES HERE"
-}
+   {
```

Comment:
are you sure the tab padding is right in this json? looks odd

Reply:
This is old JSON 🤔

Reply:
It was incorrect formatting. Thank you for pointing that out. Updated!